specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateOrUpdateADedicatedHostGroup.json

```diff
@@ -31,15 +32,17 @@
         ],
         "properties": {
           "platformFaultDomainCount": 3,
-          "hosts": []
+          "hosts": [],
```

Comment:
I don't believe we return hosts as an empty array in the PUT/PATCH response, can you pls. confirm.

Reply:
You are right.